lib/config/config.js

```diff
@@ -107,6 +107,20 @@ module.exports = function (fs, path, url, convict) {
         format: 'nat',
         env: 'IP_RATE_LIMIT_BAN_DURATION_SECONDS'
       },
+      smsRateLimit: {
+        limitIntervalSeconds: {
+          doc: 'Duration of automatic throttling for sms',
+          default: 60 * 15,
+          format: 'nat',
+          env: 'SMS_RATE_LIMIT_INTERVAL_SECONDS'
+        },
+        maxSMSs: {
```

Comment:
Heh guess a pattern that I thought made sense to me was to capitalize acronyms only if in between words. No issues changing to `maxSmss`.